spyder/preferences/shortcuts.py

```diff
@@ -291,7 +291,16 @@ def accept(self):
 
     def event(self, event):
         """Qt method override."""
-        if event.type() in (QEvent.Shortcut, QEvent.ShortcutOverride):
+        # We reroute all ShortcutOverride events to our keyPressEvent and block

```

Comment:
Why do we remove the `QEvent.Shortcut` @jnsebgosselin  ?

Reply:
It's never called... probably since we are blocking them in the override. But I'm not sure. I can add this back in the condition.

Reply:
@ccordoba12 ?

Reply:
> It's never called

If it's never called, then we should leave it out.